### DISCUSSION TITLE
How to access user and administrator guides?

### DISCUSSION THREAD

**Discussion by mfavetti** [exoplatform/exo-community#32] [Q&A]
From: https://docs.exoplatform.org/guide/userAndAdminGuide.html

"User and Functional Administrator guides are now directly shipped with the platform. You can find them in the space 'eXo Knowledge base', open to administrators."

I see no spaces at all on a new instance of 6.4 (running via docker compose) logged in as root or the first created user.

**Comment by rdenarie** [comment]
Hello,
I just seeing there is a problem with documentation import at first startup. 

You can import it manually, waiting we fix this point.
For that, you can go in notes application in a space, and follow this video :
![image](https://raw.githubusercontent.com/exoplatform/exo-documentation/kbassets/KB/FR/17812-2.gif)

English zip is located here : 
[20230905_kb_export_en.zip](https://github.com/exoplatform/exo-community/files/12524686/20230905_kb_export_en.zip)

**Comment by mfavetti** [comment]
For anyone else trying to do this, from a fresh install you need to do the following steps in order to read the guides:

1) Create a space (Menu -> Spaces -> Add space)
2) Enable the notes application for the new space (Menu -> Administration -> Spaces -> Edit button for new space -> Arrow icon next to applications -> Plus sign in applications list -> Plus sign next to notes in the sidebar that appears)
3) Go to the notes application in the new space (Menu -> Spaces -> New space -> Notes in menu bar)
4) Import the file like in the video above (you do not need to extract the archive first, the app can use it directly)
5) Refresh the page

**Reply by rdenarie** [reply]
Thanks a lot for the complete scenario !

I imagine that you manage to import notes :)
